app/src/main/java/fr/free/nrw/commons/review/ReviewActivity.java

```diff
@@ -113,7 +114,8 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         btnSkipImage.setOnClickListener(view -> {
-            reviewPagerAdapter.disableButtons();
+            reviewImageFragment=getInstanceOfReviewImageFragment();
```

Comment:
Please a space before and after `=`.
(here and in other similar lines of your PR)